BINDINGS.md

```diff
@@ -125,11 +125,13 @@ Example:
 When "Properties" autocomplete contains <string> in all elements
 
 * Select value inside dropdown:
+[Given("user selected \"(.*) in (.*)")]
 [When("user selects \"(.*)\" in (.*)")]
 Example:
 When user selects <propertyValue> in "Properties" autocomplete
 
 * Select multiple entries in dropdown with multiple choice:
+[Given("user selected multiple entries in (.*)")]
```

Comment:
Please, add : symbol to the end

Reply:
`[Given("user selected multiple entries in (.*?):")]`

Reply:
Fixed